MAUI/UIKit/GettingStarted.md

```diff
@@ -0,0 +1,81 @@
+---
+layout: post
+title: Getting Started with MAUI UI Kit control | Syncfusion
+description: Learn here about getting started with Syncfusion Essential Studio MAUI UI Kit control, its elements and more.
+platform: maui
+control: MAUI UI Kit
+documentation: ug
+---
+
+# Getting Started with MAUI UI Kit
+
+The UI Kit screens can be added in your application by the following two ways:
+
+1. Using **Essential UI Kit for MAUI** Visual Studio extension.
+
+2. Copying the files from our open source [GitHub repository](https://github.com/syncfusion/essential-ui-kit-for-.net-maui).
+
+
+## Essential UI Kit for MAUI Extension
+
+This is the easiest way to add the pre-defined screens to your application. The following steps explain how to add screens to an application with our extension: 
+
+### Extension for Windows
+
+1. Open Visual Studio.
+
+2. Go to Extension, and then click Manage Extensions as shown in the following screenshot.
+
+   ![Visual Studio Extensions](UI-Kit-images/VS_Extensions.png)
+
+3. Search for **Essential UI Kit for MAUI**, and then install it.
+
+   ![Visual Studio Extensions and Updates](UI-Kit-images/Extension_Update.png)
+
+4. Restart the Visual Studio and allow it to complete the installation. 
+
+5. Now, open an existing MAUI application or create a new application as per your requirements.
+ 
+6. Right-click the MAUI [NETStandard] project, and you can see the **Essential UI Kit for MAUI** option.
+
+N> The **Essential UI Kit for MAUI** add-in will be shown when the project have the **MAUI** NuGet package as a reference and also, MAUI project should be a NETStandard project.
+
+7. Select the category and pages you need to add in your application. In the following screenshot, the **Login with Social Icon** screen has been selected from the **Forms** category. 
+
+   ![Visual Studio UIkit Category](UI-Kit-images/Essential_UIKit_Category.png)
+8. Clicking the 'Next' button will navigate to the following page to add page name : 
+
+   ![Visual Studio UIkit Category](UI-Kit-images/Essential_UIKit_PageName.png)
+
+9. Clicking the 'Add' button will include the selected page to your project. The necessary class files, resources, and NuGet package references will automatically be added to your project as shown in the following screenshot.
+
+   ![Visual Studio Ui Kit Resources](UI-Kit-images/Essential_UIKit_Resources.png)
+   ![Visual Studio Ui Kit Resources](UI-Kit-images/Essential_UIKit_Nuget_Packages.png)
+   ![Visual Studio Ui Kit Resources](UI-Kit-images/Essential_UIKit_ClassFiles.png)
+
+## How to render the added page
+
+In a MAUI demo application, you must make the added page as the start-up page in the AppShell.xaml file. 
+Example: If you added the Login with Social Icon Page, then you must invoke the page as demonstrated in the following code.
+
+1. Add the namespace as shown in the following code:
+{% highlight xaml %}
+<Shell
```

Comment:
Dont mention in shell content

**For NET 9 project:**
![image](https://github.com/user-attachments/assets/0281ef52-8bdd-408f-b437-d1cad766a97f)

**For Net 8:**
![image](https://github.com/user-attachments/assets/8e909c9b-dba0-47d5-9188-f8b20ab1919c)

Reply:
Changed the Sample Codes as per Net 8 Sample Code in Latest Commits.